data/mods/Aftershock/items/armor/exosuit/exosuit_spells.json

```
@@ -16,5 +16,142 @@
     "difficulty": -20,
     "max_level": 1,
     "energy_source": "NONE"
+  },
+  {
+    "type": "SPELL",
+    "id": "exo_flamethrower",
+    "name": "Exosuit Flamethrower",
+    "description": "A massive gout of flame leaps forward, igniting everything in it's path.",
+    "effect": "attack",
+    "shape": "cone",
+    "valid_targets": [ "hostile", "ground" ],
+    "flags": "IGNITE_FLAMMABLE",
+    "message": "",
+    "max_level": 1,
+    "min_damage": 30,
+    "max_damage": 30,
+    "min_range": 7,
+    "max_range": 7,
+    "min_aoe": 15,
+    "max_aoe": 15,
+    "damage_type": "heat",
+    "field_id": "fd_fire",
+    "field_chance": 2,
+    "min_field_intensity": 10,
+    "max_field_intensity": 10,
+    "field_intensity_increment": 1,
+    "field_intensity_variance": 0.1,
+    "sound_description": "a loud woosh"
+  },
+  {
+    "type": "SPELL",
+    "id": "exo_sonic",
+    "name": "Exosuit Sonic Cannon",
+    "description": "Your teeth ache as the cannon fires and a ripple shoots through the air.",
+    "effect": "attack",
+    "effect_str": "stunned",
+    "shape": "cone",
+    "valid_targets": [ "hostile", "ground" ],
+    "flags": "LOUD",
+    "message": "",
+    "max_level": 1,
+    "min_damage": 30,
+    "max_damage": 30,
+    "min_range": 7,
+    "max_range": 7,
+    "min_aoe": 45,
+    "max_aoe": 45,
+    "damage_type": "bash",
+    "sound_description": "an earsplitting ring"
+  },
+  {
+    "id": "exo_laserturret",
+    "type": "SPELL",
+    "name": "Exosuit Laser Turret",
+    "description": "A burst of invisible laser pulses stab at the target.",
+    "valid_targets": [ "hostile" ],
+    "effect": "attack",
+    "shape": "line",
```

```suggestion
    "shape": "blast",
```
Looks like this one slipped through